extensions/wikia/ThemeDesigner/ThemeDesignerController.class.php

```diff
@@ -101,67 +99,59 @@ public function executePreview() {
 	private function getUploadErrorMessage( $status ) {
 		global $wgFileExtensions, $wgLang;
 
-		switch( $status[ 'status' ] ) {
+		switch ( $status['status'] ) {
 			case UploadBackgroundFromFile::FILESIZE_ERROR:
-				$msg = wfMsgHtml( 'themedesigner-size-error' );
+				$msg = wfMessage( 'themedesigner-size-error' )->escaped();
 				break;
 			case UploadBackgroundFromFile::FILETYPE_ERROR:
 			case UploadWordmarkFromFile::FILETYPE_ERROR:
-				$msg = wfMsgHtml( 'themedesigner-type-error' );
+				$msg = wfMessage( 'themedesigner-type-error' )->escaped();
 				break;
 			case UploadWordmarkFromFile::FILEDIMENSIONS_ERROR:
-				$msg = wfMsgHtml( 'themedesigner-dimensions-error' );
+				$msg = wfMessage( 'themedesigner-dimensions-error' )->escaped();
 				break;
 			case UploadBase::MIN_LENGTH_PARTNAME:
-				$msg = wfMsgHtml( 'minlength1' );
+				$msg = wfMessage( 'minlength1' )->escaped();
 				break;
 			case UploadBase::ILLEGAL_FILENAME:
-				$msg = wfMsgExt(
+				$msg = wfMessage(
 					'illegalfilename',
-					'parseinline',
 					$status['filtered']
-				);
+				)->parse();
 				break;
 			case UploadBase::OVERWRITE_EXISTING_FILE:
-				$msg =  wfMsgExt(
-					$status['overwrite'],
-					'parseinline'
-				);
+				$msg = wfMessage(
+					$status['overwrite']
+				)->parse();
 				break;
 			case UploadBase::FILETYPE_MISSING:
-				$msg = wfMsgExt(
-					'filetype-missing',
-					'parseinline'
-				);
+				$msg = wfMessage(
+					'filetype-missing'
+				)->parse();
 				break;
 			case UploadBase::EMPTY_FILE:
-				$msg = wfMsgHtml( 'emptyfile' );
+				$msg = wfMessage( 'emptyfile' )->escaped();
 				break;
 			case UploadBase::FILETYPE_BADTYPE:
 				$finalExt = $status['finalExt'];
 
-				$msg = wfMsgExt(
+				$msg = wfMessage(
 					'filetype-banned-type',
-					array( 'parseinline' ),
 					htmlspecialchars( $finalExt ),
 					implode(
-						wfMsgExt(
-							'comma-separator',
-							array( 'escapenoentities' )
-						),
+						wfMessage( 'comma-separator' )->escaped(),
 						$wgFileExtensions
 					),
 					$wgLang->formatNum( count( $wgFileExtensions ) )
-				);
+				)->parse();
 				break;
 			case UploadBase::VERIFICATION_ERROR:
 				unset( $status['status'] );
 				$code = array_shift( $status['details'] );
-				$msg = wfMsgExt(
+				$msg = wfMessage(
 					$code,
-					'parseinline',
 					$status['details']
-				);
+				)->parse();
 				break;
 			case UploadBase::HOOK_ABORTED:
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce this "switch/case" number of lines from 11 to at most 10, for example by extracting code into function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1151)